apps/prairielearn/src/lib/externalGraderLocal.ts

```diff
@@ -150,8 +177,6 @@ export class ExternalGraderLocal {
         output,
       });
 
-      results.job_id = grading_job.id;
```

Comment:
Why this change? From what I can tell, this exists to mirror the production grader:

https://github.com/PrairieLearn/PrairieLearn/blob/bb6b0324ab64e3e904df64b7a6e808aeaabaf959/apps/grader-host/src/index.js#L634

Reply:
This has been moved to the initialization of the `results` object:
https://github.com/PrairieLearn/PrairieLearn/blob/99e5fef184dc5acab82396967cc137498d704040/apps/prairielearn/src/lib/externalGraderLocal.ts#L41

Reply:
Gotcha, thanks for pointing that out.